src/components/Dancing.js

```diff
@@ -124,13 +151,16 @@ export default function Dancing({ dance, danceTime, onWebSocketClose }) {
               socket.close();
               setSocket(null);
             }
+            setDance(false);
             setTimeout(() => {
               onWebSocketClose();
-            }, 3000); // Delay of 5 seconds
+            }, 3000); // Delay of 3 seconds
```

Comment:
I dont think 3 sec will be enough for check score and high score, 5 sec is enough?

Reply:
3 sec is enouch, i test it